addon/.eslintrc.js

```diff
@@ -38,7 +38,6 @@ module.exports = {
         './testem.js',
         './blueprints/*/index.js',
         './config/**/*.js',
-        './tests/dummy/config/**/*.js',
```

Comment:
Does this file still need the "test fles" section on lines 51-58?

Reply:
@elwayman02 we indeed do not need. Also there bunch of other things that could be cleaned up, but my thought process is to get this land with minimal actual changes in file and focus on moving files into locations where they should be.

as next step, we'll sync `test-app` with latest ember-cli app blueprint and sync `addon` folder setup with https://github.com/embroider-build/addon-blueprint after we conver the actual addon to v2 format.

Sorry didn't communicate this plan before, hope this makes sense as with such approach we should have more or less easily reviewable PRs.

Reply:
Yup, makes sense!